lib/txindex.go

```diff
@@ -28,6 +29,12 @@ type TXIndex struct {
 
 	// Core params object
 	Params *BitCloutParams
+
+	// Update wait group
+	updateWaitGroup sync.WaitGroup
```

Comment:
Could we make this `UpdateWaitGroup`

Reply:
Sure! Got a tl;dr on core heuristics for public vs private? As a third party the prevalence of public fields and methods has been appreciated, but wasn’t sure if there is any obvious cut points being used to decide.

Reply:
Was just thinking making it public is more future-proof. And you mentioned you're new to Go so wasn't sure if that was intentional or not. Nonetheless, congrats on the double-merge! :)